js/stats/stats.controller.js

```diff
@@ -1,3 +1,7 @@
+// Require sub components and services
+require('./statsTotal.component.js');
+require('../version/version.component.js'); // version is currently inside stats
```

Comment:
Does this mean, we should make a separate controller for the version? Or where / how would you implement this? I think it doesn't need to be part of stats...